perceval/backends/core/mattermost.py

```diff
@@ -153,6 +153,12 @@ def fetch_items(self, category, **kwargs):
                     fetching = False
                     break
 
+                metadata = post.get('metadata', None)
+                if metadata:
+                    images = post['metadata'].get('images', None)
+                    if images:
+                        post['metadata']['images'] = self._parse_images(images)
```

Comment:
I wouldn't use `metadata` word because we have a decorator that already has that name. It shouldn't be a problem but it's better to find another name to avoid unexpected bugs. 
Also, as you don't use the variable, it's better to do:
```suggestion
                if 'metadata' in post and 'images' in post['metadata']
                    post['metadata']['images'] = self._parse_images(images)
```